ui/src/app/machines/views/AddMachine/AddMachineFormFields/AddMachineFormFields.js

```diff
@@ -124,6 +126,49 @@ export const AddMachineFormFields = () => {
           required
           type="text"
         />
+        {extraMACs.map((mac, i) => (
+          <div
+            className="p-input--closeable"
+            data-test={`extra-macs-${i}`}
+            key={`extra-macs-${i}`}
+          >
+            <Input
+              maxLength="17"
+              onChange={e => {
+                const newExtraMACs = [...extraMACs];
+                newExtraMACs[i] = e.target.value;
+                setExtraMACs(newExtraMACs);
+                setFieldValue("extra_macs", newExtraMACs);
+              }}
+              placeholder="00:00:00:00:00:00"
+              type="text"
+              value={mac}
+            />
+            <Button
+              className="p-close-input"
+              hasIcon
+              onClick={() => {
+                const newExtraMACs = extraMACs.filter((_, j) => j !== i);
+                setExtraMACs(newExtraMACs);
+                setFieldValue("extra_macs", newExtraMACs);
+              }}
+              type="button"
```

Comment:
Does this need a type?

Reply:
Yes, otherwise I get the following error from Formik:
```
Warning: You submitted a Formik form using a button with an unspecified `type` attribute
Most browsers default button elements to `type="submit"`.
If this is not a submit button, please add `type="button"`
```